### PR TITLE
feat: Get follower ids

### LINKED ISSUE #275: Support get follower IDs api
Doc: https://developers.line.biz/ja/reference/messaging-api/#get-follower-ids

### PR DESCRIPTION
**Pull request by louis70109**
Closed #275

### CODE REVIEW

File: lib/client.ts [hunk 1]
```diff
@@ -200,6 +200,26 @@ export default class Client {
     return this.getChatMemberIds("room", roomId);
   }
 
+  private async getBotFollowers(): Promise<string[]> {
```
Comment (xingoxu) [inline lib/client.ts:203]: You don't have to set this method to private. Just use public.

**Review by xingoxu — CHANGES_REQUESTED** [review]

File: lib/client.ts (hunk 1 above)
Comment (louis70109) [reply lib/client.ts]: Fixed, thanks!

File: lib/client.ts [hunk 2]
```diff
@@ -200,6 +200,26 @@ export default class Client {
...
+      userIds = userIds.concat(res.userIds);
+      start = res.next;
+    } while (start);
+
+    return userIds;
+  }
+  public async getBotFollowersIds(): Promise<string[]> {
+    return this.getBotFollowers();
+  }
+
```
Comment (xingoxu) [inline lib/client.ts:222]: so why we wrap this again? Just 
```
public async getBotFollowersIds() {
   // contents
}
```

**Review by xingoxu — CHANGES_REQUESTED** [review]

**Comment by taichunmin** [comment]
LGTM

**Comment by 4geru** [comment]
@xingoxu excuse me. this pull request seems lgtm. please review it :bow:

**Pull request merged**